pkg/rpcclient/rpc.go

```diff
@@ -1042,6 +1048,9 @@ func (c *Client) CalculateValidUntilBlock() (uint32, error) {
 
 // AddNetworkFee adds network fee for each witness script and optional extra
 // network fee to transaction. `accs` is an array signer's accounts.
+//
+// Deprecated: please use CalculateNetworkFee or actor.Actor. This method will
+// be removed in future versions.
 func (c *Client) AddNetworkFee(tx *transaction.Transaction, extraFee int64, accs ...*wallet.Account) error {
```

Comment:
Somehow I had a thought that `AddNetworkFee` differs a bit from `CalculateNetworkFee` in case of the contract-based witness. But now after the code review they look identical. Just in case, are you sure we want to remove this method from the client?

Reply:
Yes, it was created before `CalculateNetworkFee` appeared and I don't like what it's doing (as well as its `accs` interface). It also performs more network requests than `CalculateNetworkFee`.